docs/subsystems/performance.md

```diff
@@ -34,7 +34,7 @@ of load profiles:
   etc. have large numbers of users, many of whom are idle. (Many of
   the others likely stopped by to ask a question, got it answered, and
   then didn't need the community again for the next year). Our own
-  [chat.zulip.org](https://zulip.com/developer-community/) is a good
+  [Zulip development community](https://zulip.com/developer-community/) is a good
```

Comment:
This line should be wrapped similar to the surrounding text as well.

Reply:
I have wrapped it around 80 characters but I'm not sure where I'm going wrong?